liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -172,15 +181,21 @@
     @Parameter(readonly = true, required = true, defaultValue = "${project.build.testOutputDirectory}")
     private File testOutputDirectory;
 
+    /**
+     * Additional options for the docker run command when dev mode starts a container.
+     */
+    @Parameter(property = "dockerRunOpts")
+    private String dockerRunOpts;
```

Comment:
I wasn't sure where to put this question. Is anything monitoring for changes in `dockerRunOpts`, `container` or `dockerfile` while running in dev mode? Where is that handled?

Reply:
Currently we don't handle changing those while running dev mode.

Reply:
Probably should add that to the documentation for the devc goal then. Do you plan on checking for changes in the future for GA?

Reply:
Added a note in the doc. No current plans to check for changes there since it's integral to how dev mode is running (particularly for the container option), so dev mode should be restarted.